### ISSUE TITLE
Install failed: "The package could not be installed. No valid plugins were found."

### ISSUE THREAD

**Issue by KaHooli** [helgatheviking/Nav-Menu-Roles#12]
I've just tried to install this plugin via the Plugin Installer on my Wordpress site, and got this error...

> Installing Plugin: Nav Menu Roles
> Downloading install package from https://downloads.wordpress.org/plugin/nav-menu-roles.zip…
> Unpacking the package…
> Installing the plugin…
> The package could not be installed. No valid plugins were found.
> Plugin install failed.
> Return to Plugin Installer

Other plugin's seem install fine.

**Comment by helgatheviking** [comment]
Looks like I borked up the SVN repo pretty good. I hate SVN so much sometimes. I just committed version 1.7.4 which (cross your fingers) will resolve it eventually. 

1.7.4 tag is now available: http://plugins.svn.wordpress.org/nav-menu-roles/tags/
which it wasn't before even though the readme.txt said 1.7.4 was the version number. I think this mis-match is what caused the problem. I'll check that it is resolved tomorrow.
